frontend/src/styles/tokens.css

```
@@ -82,7 +100,8 @@
      폰트 (Fonts)
      ============================================ */
 
-  --font-sans: 'Pretendard', 'Apple SD Gothic Neo', sans-serif;
+  --font-sans: 'SF Pro Text', 'Pretendard', system-ui, -apple-system, BlinkMacSystemFont, sans-serif;
```

Stylelint (17.9.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)